lib/faker/util.ex

```diff
@@ -20,14 +20,22 @@ defmodule Faker.Util do
       "c"
       iex> Faker.Util.pick([1, "2", 3.0])
       "2"
+      iex> Faker.Util.pick(%{"state1" => "city1", "state2" => "city2"})
+      {"state1", "city1"}
   """
-  @spec pick(Enum.t()) :: any
-  def pick(enum) do
-    enum
-    |> Enum.to_list()
-    |> Enum.at(Faker.random_between(0, Enum.count(enum) - 1))
+  @spec pick([any]) :: any
+  def pick(list) do
+    Enum.at(list, Faker.random_between(0, Enum.count(list) - 1))
```

Comment:
This perfomance issue was already fixed https://github.com/igas/faker/pull/221

Reply:
Sorry, I didn't realize that if I added more commits to master on my fork after the approval those commits would still be included in the pull request.
What should I do now? Maybe I'll start again from 0.11.2, make a branch, add the Italian modules, create a new pull request from there and forget about this one. Would that be ok?

Reply:
Just update this branch with master and we'd be okay.

Reply:
In other words, fix the merge issues.

Reply:
Done.